jcstress-core/src/main/java/org/openjdk/jcstress/vm/VMSupport.java

```diff
@@ -334,15 +334,19 @@ public static void detectAvailableVMConfigs(boolean splitCompilation, List<Strin
             configs = newConfigs;
         }
 
-        // Mix in input arguments, if available
-        List<String> inputArgs = new ArrayList<>();
-
+        // Mix in input arguments, if available, skipping the debug options
+        List<String> mxArgs;
         try {
-            inputArgs.addAll(ManagementFactory.getRuntimeMXBean().getInputArguments());
+            mxArgs = ManagementFactory.getRuntimeMXBean().getInputArguments();
         } catch (InternalError e) {
```

Comment:
not sure where is `InternalError` coming from , do we need to catch it ? why are we not re-throwing it since its a sub-class of `Error` ?
not able to find the source code of `sun.management.spi.PlatformMBeanProvider.PlatformComponent` , can you help me with it ..!??!